### DISCUSSION TITLE
iFrame suggestions

### DISCUSSION THREAD

**Discussion by tmoindustries** [tokenbound/docs#70] [Ideas]
love the iFrame feature.  A few ideas:
- make sure th image fills the whole box.  ours is 500x500 and is no reaching the edges on the marketplaces but is on TBA site. Even the demo iframe Sapienz in your docs does not fill the whole frame
- show quantity of 1155's.  TBA site does this.
- click through to nfts. TBA allows this.  Maybe marketplaces are disabling.

Thanks for all you are doing!

**Comment by tmoindustries** [comment]
screen recording
https://share.descript.com/embed/hxe8GyEoqRF

**Comment by huynhr** [comment]
@tmoindustries Thanks for putting this together. I love these suggestions. I can push up a few changes to support the image taking the full width and quantities of 1155s sometime this week. 

Unfortunately OS disables some iframe interactions such as clicking through to the NFT's and the etherscan link out. I've added it in to show the OS team that it isn't working, but still waiting for them to get back to me. 

Also, the iframe is open source so if you feel like there's a feature you want in there please feel free to open a PR and we can review those features as well :).

**Reply by huynhr** [reply]
@tmoindustries  Iframe has been updated for the requested changes:

https://github.com/tokenbound/iframe/pull/31

Closing this discussion as changes have been merged.

**Reply by tmoindustries** [reply]
one more: for Eth balances in iframe anything under 0.00 does not show - need to add some more 0's especially for L2 where lower bal are very common. Thanks!

@huynhr
